lms/product/plugin/misc.py

```diff
@@ -40,24 +40,10 @@ def get_ltia_aud_claim(self, lti_registration: LTIRegistration) -> str:
         """Get the value of the `aud` claim used in LTI advantage requests."""
         return lti_registration.token_url
 
-    def get_document_url(self, request):
+    def get_document_url(self, _request, assignment, historical_assignment):
```

Comment:
In the general case get if from the assignment or the copy.

Not sorter code but note that we remove one place to query for the copy and assignment. I reckon having only place doing that is much easier to follow.